src/ripple/basics/TaggedCache.h

```diff
@@ -61,7 +61,6 @@ class TaggedCache
     using mutex_type = Mutex;
     // VFALCO DEPRECATED The caller can just use std::unique_lock <type>
     using ScopedLockType = std::unique_lock <mutex_type>;
```

Comment:
Perhaps in a future PR it would be good to remove the `ScopedLockType` typedef from this class.

Reply:
Agreed. Will fix in this PR.

Reply:
Fixed in 3fd690eae "[fold] Rm ScopedLockType and use std::unique_lock directly"